code/common/utils.q

```diff
@@ -67,8 +67,10 @@ gettableproperty:extractfromsubdict[;`tableproperties;];   //- extract from `tab
 
```

Comment:
Can you rename this file please as part of this- we shouldn't have code/common/utils.q (utils for what?) It needs a propername, or if this is entirely related to the dataaccess stuff, can it live in the dataaccess directory?